src/Business/Detection/Common/Repositories/IDetectionRepository.cs

```diff
@@ -9,5 +9,9 @@ public interface IDetectionRepository
         public IEnumerable<MDetection> ByParameter();
 
         public IEnumerable<MDetection> Insert(IEnumerable<MCreateDetection> input);
+
+        public MDetection Insert(MCreateDetection input);
```

Comment:
Should we delete this method and use the other Insert which will insert a list of inputs?